src/packages/views.py

```diff
@@ -71,9 +109,9 @@ def partner_view(request):
     return render(request, "packages/partner.html", context)
 
 
-def partner_new_order(request):
-    partner = Partner.objects.get(name=request.GET.get('p'))
-    order = Order.objects.get(id=request.GET.get('order'))
+def partner_order_view(request, partner, order):

```

Comment:
```python
def partner_order_view(request, partner_name, order_id)
```

Reply:
Yeah, I went ahead and fixed all occurrences of this inconsistency that I've found after the first comment